runtime/src/impls.rs

```diff
@@ -35,6 +36,45 @@ use sp_runtime::{
 	DispatchError, Fixed64,
 };
 
+pub type NegativeImbalance = <<Runtime as crml_transaction_payment::Trait>::Currency as Currency<
+	<Runtime as frame_system::Trait>::AccountId,
+>>::NegativeImbalance;
+pub type PositiveImbalance = <<Runtime as crml_transaction_payment::Trait>::Currency as Currency<
+	<Runtime as frame_system::Trait>::AccountId,
+>>::PositiveImbalance;
```

Comment:
This can be removed if we define them in `runtime/src/lib.rs` like this:
```rust
type PositiveImbalance = <GenericAsset as MultiCurrencyAccounting>::PositiveImbalance;
type NegativeImbalance = <GenericAsset as MultiCurrencyAccounting>::NegativeImbalance;
```
and import the above using `use crate::{PositiveImbalance, NegativeImbalance};`

Reply:
done